.gitignore

```diff
@@ -46,6 +46,9 @@ build/Release
 node_modules/
 jspm_packages/
 
+# typescript build from compiling
+built/
```

Comment:
Are you sure the correct name for this directory is `built`?
I think usually the output directory is called `build`, so with a `d` at the end?

Reply:
I just use conventions from work that I do at my job. and we call the folder `built`. I can make the change though, no problem.